### ISSUE TITLE
Cross Compiling Error

### ISSUE THREAD

**Issue by xaya** [xaya/xaya#57]
for part of the cross compile on Ubuntu (using this guide https://github.com/bitcoin/bitcoin/blob/master/doc/build-windows.md ) you must run :

~/xaya/depends$ make HOST=x86_64-w64-mingw32
or
~/xaya/depends$ make HOST=x86_64-apple-darwin11

but an error is returned

make: *** No targets specified and no makefile found. Stop.

**Comment by domob1812** [comment]
Indeed, in Xaya there are files removed in `depends/` compared to Namecoin (while there are no changes at all between Namecoin and Bitcoin).  Not sure how that happened, but I'll fix it.
